### PR TITLE
Miscellaneous libultra header fixes

### PR DESCRIPTION
**Pull request by cadmic**
The rest of #2062.

* Fix type of osTvType etc.
* Remove macros OS_SEC_TO_CYCLES and OS_MSEC_TO_CYCLES and add comments if there weren't already
* Replace VI_CTRL_PIXEL_ADV(3) with VI_CTRL_PIXEL_ADV_3
* Add FloatInt for rand.c to avoid using libultra-internal fu union

### CODE REVIEW

File: src/code/graph.c [hunk 1]
```diff
@@ -179,6 +179,7 @@ void Graph_TaskSet00(GraphicsContext* gfxCtx) {
     {
         CfbInfo* cfb;
 
+        // Schedule 3 second RCP timeout
```
Comment (Dragorn421) [inline src/code/graph.c:182]:
```suggestion
        // Schedule RCP timeout after 3 seconds
```
?

Comment (cadmic) [reply src/code/graph.c:182]: hmm makes it sound like the scheduling happens in 3 seconds instead of now

Comment (krm01) [reply src/code/graph.c:182]: `after` -> `for` ?

Comment (cadmic) [reply src/code/graph.c:182]: that would address that concern, but what's wrong with "Schedule 3 second RCP timeout"?

Comment (cadmic) [reply src/code/graph.c]: I borrowed the wording from some other places OS_USEC_TO_CYCLES is used for this purpose

**Review by Dragorn421 — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

**Pull request merged**